File: 0633-sum-of-square-numbers/0633-sum-of-square-numbers.py
class Solution(object):
    def judgeSquareSum(self, c):
        """
        :type c: int
        :rtype: bool
        """
        values = set()
        i = 0
        while i**2 <= c:
            values.add(i**2)
            i += 1
        print(values)
        for v in values:
            check = c - v
            if check in values:
                return True
        return False

File: 0633-sum-of-square-numbers/0633-sum-of-square-numbers.java
class Solution {
    public boolean judgeSquareSum(int c) {
        int count = 0; 
        while(Math.pow(count, 2) <= c){
            int sq = (int)Math.sqrt(c - ((int)Math.pow(count, 2)));
            if(Math.sqrt(c-((int)Math.pow(count, 2))) == sq ){
                return true;
            }
            count++;
        }
        
        return false;
        
    }
    
}